src/System.IO.Compression/tests/ZipArchive/zip_ManualAndCompatibilityTests.cs

```diff
@@ -21,6 +20,13 @@ public class zip_ManualAndCompatabilityTests : ZipFileTestBase
             IsZipSameAsDir(await StreamHelpers.CreateTempCopyStream(compat(zipFile)), zfolder(zipFolder), ZipArchiveMode.Update, requireExplicit, checkTimes);
         }
 
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Deflate64 zip support is a netcore feature not available on full framework.")]
+        public static async Task Deflate64Zip()
+        {
+            IsZipSameAsDir(await StreamHelpers.CreateTempCopyStream(compat("deflate64.zip")), zfolder("normal"), ZipArchiveMode.Update, requireExplicit: true, checkTimes: true);
```

Comment:
Nit: rather than duplicating the body of the test, you can make it just be:
```C#
await CompatibilityTests("deflate64.zip", "normal", true, true);
```